pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -6588,10 +6588,6 @@ literal:
 |   HEXNUM
     {
         switch v := $1.(type) {
```

Comment:
change HEXNUM as %type<str> and remove switch.

Reply:
ok, it's easy to modify